src/views/SettingsPersonal.tsx

```diff
@@ -0,0 +1,511 @@
+                                            label={
+                                            <Text style={{color: "gray"}}><b style={{color: "black"}}>Name</b> (Set a name to identify the token.)</Text>
+                                            }
+                                            rules={[{
+                                                required: true,
+                                                message: 'Please add a name for this personal access token',
+                                                whitespace: true
+                                            }]}
+                                        >
+                                            <Input
+                                                placeholder={""}
+                                                ref={inputNameRef}
+                                                autoComplete="off"/>
+                                        </Form.Item>
+                                    </Col>
+                                </Row>
+                                <Divider style={{marginTop: "0px"}}></Divider>
+                            </Col>
+                            <Col span={24} style={{textAlign: "left"}}>
+                                <Form.Item
+                                    name="expires_in"
+                                    label={
+                                        <Text style={{color: "gray"}}><b style={{color: "black"}}>Expires</b> (Set the amount of days the token should be valid.)</Text>
+                                    }
+                                    rules={[{
+                                        type: 'number',
+                                        min: 1,
+                                        max: 356,
+                                        message: 'The expiration should be set between 1 and 365 days'
+                                    }]}>
+                                    <InputNumber/>
+                                </Form.Item>
+                            </Col>
+                            <Col span={24} style={{textAlign: "center"}}>
+                                <Divider style={{marginTop: "0px"}}></Divider>
+                                <Button icon={<QuestionCircleFilled/>} type="link" target="_blank" disabled={true}
+                                        href="https://netbird.io/docs/overview/personal-access-tokens">Learn more about personal access tokens</Button>
+                            </Col>
+                        </Row>
+                    </Form>}
+                    {showPlainToken && <Space className="nb-code" direction="vertical" size="middle">
+                        <Row>
+                            <>
+                                <Text style={{fontSize: "large", marginTop: "-4px", backgroundColor: "#ebeef3", padding: "10px", borderRadius: "5px", marginBottom: "20px"}}>{plainToken}</Text>
```

Comment:
I think that it makes sense to use the same component - SyntaxHighlighter as in TabSteps.tsx

Reply:
switched to SyntaxHighlighter